### PR TITLE
Expose v2 current user api

### PR DESCRIPTION
**Pull request by islemaster**
Simple endpoint for an app to get the current user's name and ID.  Kept this really thin for now, but we might reasonably expand what this returns within reason, as need arises.

### CODE REVIEW

File: pegasus/helpers/section_api_helpers.rb [hunk 1]
```diff
@@ -6,6 +6,15 @@ def self.admin?(user_id)
   end
 end
 
+class DashboardUser
+  def self.to_self_view(user_row)
```
Comment (laurelfan) [inline pegasus/helpers/section_api_helpers.rb]: Do we have Hash#slice in pegasus?
eg: `user_row.slice(:id, :name)`

Comment (laurelfan) [inline pegasus/helpers/section_api_helpers.rb]: it looks like you were considering calling this to_self_hash... any particular rationale for calling this filtering method to_self_view and the other ones in Section to_owner_hash/etc?

Comment (islemaster) [inline pegasus/helpers/section_api_helpers.rb]: It felt like the name was getting less and less descriptive... `to_self_hash` on a hash sounds like a no-op.  Also, this one's a static method.  So knowing that `slice` exists, maybe I should ditch this method altogether and just call slice in the route definition.

**Comment by islemaster** [comment]
So, how about this, the super-minimal way?

File: pegasus/helpers/section_api_helpers.rb (hunk 1 above)
Comment (wjordan) [inline pegasus/helpers/section_api_helpers.rb]: [Hash#slice](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/hash/slice.rb#L21-L24) looks to be a Rails-only extension.

**Comment by laurelfan** [comment]
ha, fancy, nice.

**Comment by wjordan** [comment]
If `.slice` is a Rails-only extension, then this will work in our current development setup, but will likely break when the Pegasus code is loaded as a separate module outside of the Rails application framework.

**Comment by islemaster** [comment]
Is `v2_user_routes.rb` ever going to run outside of a Rails context?

**Comment by laurelfan** [comment]
yes... ah... pegasus has a slice_keys method that does the same thing as slice

**Comment by islemaster** [comment]
Okay, now using slice_keys.

**Pull request merged**